proposed/extended-coding-style-guide.md

```diff
@@ -896,7 +896,7 @@ if ($a === $b) {
 
 ## 7. Closures
 
-Closures MUST be declared with a space after the `function` keyword, and a
+Closures MUST be declared with space after the `function` keyword, and a
```

Comment:
IMO, this makes this sentence less clear - `with a space` = precisely one space, `with space` could mean any number of spaces, tabs or even new lines.

Reply:
Same objection that I made: https://github.com/php-fig/fig-standards/pull/1161#discussion_r270668995

But since it's a negation (`MUST NOT`) it doesn't matter how many, no spaces are allowed.

Reply:
@jean85, I don't follow what you mean...

This line is a `MUST`, not a `MUST NOT`...

Without the `a`, it does read as "MUST have Greater Than Zero spaces"... thus, 20 spaces after `function` is valid.

Reply:
Umh you're right, my comment was on another line!

Reply:
This case, indeed, may cause issues. I'll send a pull request.

Reply:
Here we are: https://github.com/php-fig/fig-standards/pull/1163